src/main/java/lotto/model/Prize.java

```diff
@@ -0,0 +1,43 @@
+package lotto.model;
+
+import java.util.EnumMap;
+import java.util.Objects;
+
+import static lotto.model.LottoMachine.LOTTO_PER_MONEY;
+
+public class Prize {
+    private final EnumMap<Rank, Integer> prizeMap;
```

Comment:
변수의 타입은 인터페이스인 Map으로 선언하면 어떨까요?
```suggestion
public class Prize {
    private final Map<Rank, Integer> prizeMap;
```
---

>EnumMap을 처음 학습 테스트하고, 반영했는데 코드가 달라져서 놀랐습니다🤔. 이러한 자료 구조는 필요할 때마다 직접 찾아보시고 하나씩 알아 가신 걸까요? 실무에서 자주 사용하시는 자료구조 또 있으실까요?

모든 API를 알 수 없으니 필요할 때 마다 찾아보는 편입니다 😃 
Enum과 관련해서 EnumSet 도 자주 쓰는 것 같아요

Reply:
"구현체가 아닌 추상화된 인터페이스에 의존해야 한다"를 
잊고 있었네요 감사합니다